### PR TITLE
Bump devnet version to v0.2.0-rc.1

### PR DESCRIPTION
**Pull request by kkawula**
Closes #
Because of problems with installing devnet on windows, devnet version was bumped to v0.2.0-rc.1, this version introduced a fix and devnet started installing properly. Due to update there was a problem with tests because there was a redeclaration of a few contracts and [it isn't further supported](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/547), these tests are marked to skip. Marks should be deleted in the future -  addressed issue #1419 

Reference to working state - [gh actions](https://github.com/kkawula/starknet.py/actions/runs/10220056363)

## Introduced changes

- Updated devnet_abortBlock
- Added marks to skip tests (#1419)

##

- [ ] This PR contains breaking changes

### CODE REVIEW

File: starknet_py/tests/e2e/client_devnet/time_test.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 @pytest.mark.asyncio
 async def test_set_and_increase_time(devnet_client):
     time = 3384617820
-    increase_value, error_margin = 100, 2
+    increase_value, error_margin = 100, 10
```
Comment (kkawula) [inline starknet_py/tests/e2e/client_devnet/time_test.py:7]: Windows is too slow for 2 ms margin

File: starknet_py/devnet_utils/devnet_client.py [hunk 2]
```diff
@@ -142,19 +146,24 @@ async def create_block(self) -> str:
...
+        self,
+        block_hash: Optional[Union[Hash, Tag]] = None,
+        block_number: Optional[Union[int, Tag]] = None,
+    ) -> List[str]:
         """
         This functionality allows simulating block abortion that can occur on mainnet.
         It is supported in the `--state-archive-capacity full` mode.
 
-        :param starting_block_hash: The state of Devnet will be reverted to the state before `starting_block_hash`.
+        :param starting_block_id: The state of Devnet will be reverted to the state before `starting_block_hash`.
```
Comment (ddoktorski) [inline starknet_py/devnet_utils/devnet_client.py:158]: It should have params for `block_hash` and `block_number` now.

File: starknet_py/tests/e2e/account/account_test.py [hunk 3]
```diff
@@ -123,6 +123,8 @@ async def test_account_estimate_fee_for_declare_transaction(
     )
 
 
+# TODO (#1419): Fix contract redeclaration
+@pytest.mark.skip(reason="Redeclaration occurred")
```
Comment (ddoktorski) [inline starknet_py/tests/e2e/account/account_test.py:127]: We can modify this test to estimate fee for a different transaction than declare and that will be sufficient.

File: starknet_py/tests/e2e/block_test.py [hunk 4]
```diff
@@ -23,6 +23,8 @@ async def declare_contract(account: BaseAccount, compiled_contract: str):
     await declare_result.wait_for_acceptance()
 
 
+# TODO (#1419): Fix contract redeclaration
+@pytest.mark.skip(reason="Redeclaration occurred")
```
Comment (ddoktorski) [inline starknet_py/tests/e2e/block_test.py:27]: We can remove `declare_contract` from this test, as it's not needed.

File: starknet_py/tests/e2e/block_test.py [hunk 5]
```diff
@@ -32,6 +34,8 @@ async def test_pending_block(account, map_compiled_contract):
     assert isinstance(blk, PendingStarknetBlock)
 
 
+# TODO (#1419): Fix contract redeclaration
+@pytest.mark.skip(reason="Redeclaration occurred")
```
Comment (ddoktorski) [inline starknet_py/tests/e2e/block_test.py:38]: The same comment as above.

**Review by ddoktorski — APPROVED** [review]

**Pull request merged**